docs/user/device-groups.md

```diff
@@ -0,0 +1,43 @@
+---
+navTitle: Device Groups
+---
+
+# Device Groups
+
+
+**Navigation**: `Team > Application > Device Groups`
+
+Device groups allow you to organise your Application assigned devices into logical groups.
```

Comment:
I believe the _why_ a user wants this isn't well covered.

When managing many devices running the same snapshot, ..... etc etc